src/NServiceBus.Testing.Fakes/TestablePipelineContext.cs

```diff
@@ -42,8 +47,15 @@ public TestablePipelineContext(IMessageCreator messageCreator = null)
         /// <param name="options">The options for the send.</param>
         public virtual Task Send(object message, SendOptions options)
         {
-            sentMessages.Enqueue(new SentMessage<object>(message, options));
+            var headers = options.GetHeaders();
+
+            if (headers.ContainsKey(Headers.IsSagaTimeoutMessage) && 
+                headers[Headers.IsSagaTimeoutMessage] == bool.TrueString)
```

Comment:
Checking the value might be redundant since it looks like we only ever set the header with a `true` value, and we have other code that just looks for the [existence of the header](https://github.com/Particular/NServiceBus/blob/b152c6837fd5dd9768c818a05add617bd7d33d48/src/NServiceBus.Core/Sagas/SagaPersistenceBehavior.cs#L215).